action_name: admittance
kinematic_chain_base_link: yumi_base_link
left_arm:
  kdl_eef_frame: yumi_link_7_l
  gripping_frame: yumi_link_7_l
  has_ft_sensor: true
  sensor_frame: optodaq_l_sensor_link
  sensor_topic: /yumi/optodaq_l_compensated

right_arm:
  kdl_eef_frame: yumi_link_7_r
  gripping_frame: yumi_link_7_r
  has_ft_sensor: true
  sensor_frame: optodaq_r_sensor_link
  sensor_topic: /yumi/optodaq_r_compensated

kdl_manager:
  eps: 0.001
  max_tf_attempts: 10

admittance:
  settling_time: 10
  damping_ratio: 1
  K_p:
    data: [5, 0, 0, 0, 0, 0,
           0, 5, 0, 0, 0, 0,
           0, 0, 5, 0, 0, 0,
           0, 0, 0, 0.1, 0, 0,
           0, 0, 0, 0, 0.1, 0,
           0, 0, 0, 0, 0, 0.1]

  force_dead_zone: 1
  torque_dead_zone: 0.2
